client/components/select-dropdown/index.jsx

```diff
@@ -132,6 +134,17 @@ class SelectDropdown extends Component {
 		return get( find( options, { value: selected } ), 'icon' );
 	}
 
+	getSelectedSecondaryIcon() {
+		const { options, selectedSecondaryIcon } = this.props;
+		const { selected } = this.state;
+
+		if ( selectedSecondaryIcon ) {
+			return selectedSecondaryIcon;
+		}
+
+		return get( find( options, { value: selected } ), 'secondaryIcon' );
```

Comment:
We're slowly trying to remove lodash, non-blocking comment but this should probably be something like 
```suggestion
		return options.find(option => option.value === 'selected')?.secondaryIcon;
```

Reply:
Makes sense, but I can't readily test that as it relies on the `options` array, which is not currently used in this case.

The Lodash implementation is the same as the other props, so it's not a big deal to keep it here, since we are not adding new Lodash imports. 🙂